### DISCUSSION TITLE
GitHub's Badges

### DISCUSSION THREAD

**Discussion by ArianDJ** [community/community#114583] [Other Feature Feedback, Questions, & Ideas]
What are all those badges?

**Comment by ArianDJ** [comment]
You got probaly a few hundred of them, when I look at the source code that is leaked in 2010, it's around 350, whereof 340 custom badges for the crown developers

**Comment by ghostinhershell** [comment]
Thanks so much for posting in our Community @ArianDJ. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀
